blog/2023-05-02-release-notes.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Luigi v2.2
+seoMetaDescription: Release notes for Luigi v2.2
+author:
+  - Aleksandra Simeonova
+layout: blog
+---
+
+You can read about the new features in Luigi v2.2 in the release notes below.
+
+<!-- Excerpt -->
+
+#### setViewGroupData function
+
+A new `setViewGroupData` function was added to the [Luigi Client API](https://docs.luigi-project.io/docs/luigi-client-api). It allows you to conveniently change node labels within the same [view group](https://docs.luigi-project.io/docs/navigation-advanced?section=view-groups). In your Luigi node configuration, you can enable it with e.g. `label: 'my Node {viewGroupData.vg1}'` and use the function to add a label: `LuigiClient.setViewGroupData({'vg1':' Luigi rocks!'})`. You can find more information in the [documentation](https://docs.luigi-project.io/docs/luigi-client-api/?section=setviewgroupdata).
+
+#### TabNav header micro frontend
```

Comment:
Might be worth discussing if we want this declared here, or we wait and publish later. Not sure.